bom/openhab-addons/pom.xml

```
@@ -1131,6 +1131,10 @@
       <artifactId>org.openhab.binding.upnpcontrol</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <artifactId>org.openhab.binding.upb</artifactId>
```

Adding the `groupId` here will help with getting the build fixed:

```suggestion
    <dependency>
      <groupId>org.openhab.addons.bundles</groupId>
      <artifactId>org.openhab.binding.upb</artifactId>
```